TVCEXECUTIVE.s

```diff
@@ -100,7 +100,7 @@ ROLLPREP	CAE	CDUX		# UPDATE ROLL LADDERS (NO NEED TO RESTART-
 		XCH	OGANOW		# 	PROTECT, SINCE ROLL DAPS RE-START)
 		XCH	OGAPAST
 		
-		CAE	OGAD		# PREPARE ROLL FDAI NEEDLE WTIH FLY-TO
```

Comment:
But, but are you sure that won't break the code. These typos must be dealt wtih more carefully.